portfolio/src/main/webapp/script.js

```diff
@@ -17,10 +17,10 @@
  */
 
  //random image generator 
-window.onload = function(){ 
- document.getElementById("rFrontTile").addEventListener("mouseover", 
+window.onload = function SetImageOnMouseOver(){ 
+ document.getElementById("rand-front-tile").addEventListener("mouseover", 
  function() {
-  getRandomFile(document.querySelector("#rBackTile img"), "images/random/", "test", 4)
+  getRandomFile(/* element= */ document.querySelector("#rand-back-tile img"), /* path= */ "images/random/", /* fileName= */ "test", /* numElems= */ 4)
```

Comment:
Line length: https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Reply:
I'll look into a plugin to ensure this is done automatically

Reply:
looks like rewrap is the vs code plugin that does what we are looking for. You have to explicitly select the line that is too long, but I think there's a way to highlight such areas so I'll know to do that. For the HTML there are a couple lines over 80, but the doc says only rewrap to improve readability so I'll leave them as is.